test/integration/agent/openflow_test.go

```diff
@@ -84,6 +88,9 @@ type testConfig struct {
 }
 
 func TestConnectivityFlows(t *testing.T) {
+	// Initialize ovs metrics (prometheus) to test them
+	metrics.InitializeOVSMetrics()
```

Comment:
I see that we call `metrics.InitializeOVSMetrics()` in multiple tests. I assume there is no issue with calling the function multiple times?

Reply:
Yes, I made sure by calling this multiple times in same test. If the metric is already registered, the registry ignores subsequent calls.